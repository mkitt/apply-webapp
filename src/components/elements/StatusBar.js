// @flow
import glamorous from 'glamorous'

const rules = {
  height: 4,
  backgroundColor: '#4d4d4d',
}

export default glamorous.div(
  rules,
  ({ children, ...props }) => ({ ...props }),
)
