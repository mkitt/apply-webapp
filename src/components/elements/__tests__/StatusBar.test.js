import React from 'react'
import renderer from 'react-test-renderer'
import StatusBar from '../StatusBar'

describe('<StatusBar />', () => {
  it('shows the correct snapshot tree for the <StatusBar> element', () => {
    const component = <StatusBar />
    const tree = renderer.create(component).toJSON()
    expect(tree).toMatchSnapshot()
  })
})
