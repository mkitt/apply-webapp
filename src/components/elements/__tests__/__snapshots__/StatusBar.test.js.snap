// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<StatusBar /> shows the correct snapshot tree for the <StatusBar> element 1`] = `
.css-0,
[data-css-0] {
  height: 4px;
  background-color: #4d4d4d;
}

<div
  className="css-0"
/>
`;
